src/utils/generators.mjs

```diff
@@ -12,14 +14,20 @@ export const groupNodesByModule = nodes => {
   /** @type {Map<string, Array<ApiDocMetadataEntry>>} */
   const groupedNodes = new Map();
 
+  const progressBar = createProgressBar(groupNodesByModule.name);
+  progressBar.start(nodes.length, 0);
```

Comment:
I don't think this should be inside this util :)

Reply:
To clarify the only thing this util should do is to group nodes by module; This is breaking the single responsibility pattern.

Reply:
Humm yeah